src/js/components/Pagination/Pagination.js

```diff
@@ -194,24 +208,34 @@ const Pagination = forwardRef(
     return (
       <>
         {children && currentItems.map(item => children(item))}
-        <Nav
-          a11yTitle={a11yTitle || 'Pagination Navigation'}
-          ref={ref}
-          {...rest}
-        >
-          <Box
-            as="ul"
-            align="center"
-            direction="row"
-            gap="xxsmall"
-            pad="none"
-            margin="none"
+        {/* Do we want pontential for internally displaying a "results summary"?
+         If so, applying the container props to this box now may leave better
+          flexibility moving forward
+        */}
+        <StyledPaginationContainer {...theme.pagination.container}>
+          {/* Ideation: internal control of some results summary, need to think 
+          about what the default would be and how it may be customized
+          */}
+          {/* {showSummary && <Text>Showing 1-10 of 500 results</Text>} */}
+          <Nav
+            a11yTitle={a11yTitle || 'Pagination Navigation'}
+            ref={ref}
+            {...rest}
```

Comment:
Should this `{...rest}` be moved up to the `<StyledPaginationContainer />`? 
```
<StyledPaginationContainer {...theme.pagination.container} {...rest} >
```

Reply:
Is that what comment on line 212 is referencing? If so, let's go ahead and make that adjustment.

Reply:
Great, done.